Nef_3/include/CGAL/Nef_3/SNC_io_parser.h

```diff
@@ -645,7 +646,7 @@ struct find_minimal_sface_of_shell : public SNC_const_decorator<T> {
     : Base(D), Done(Vi), SORT(D) {}
 
   void visit(SFace_handle h) {
-    Done[h]=true;
+    Done.insert(h);
```

Comment:
It seems the values are only ever inserted into the hashmap here, but never read?!
I think we can remove the `Done` hashmap entirely in this file.

Reply:
I was wrong.  `visit_shell_objects()` calls `visit()`.

Reply:
It is also a possibility that if `Done[h] == true` we might immediately return from `visit()`.

Reply:
@afabri Perhaps this is a bug (or at least a separate enhancement). It might already be something @maxGimeno has changed in his IO improvements work.

Reply:
I don't think so, it doesn't ring any bell.

Reply:
It's like this for quite some time I would say, when we [blame](https://github.com/CGAL/cgal/blame/master/Nef_3/include/CGAL/Nef_3/SNC_io_parser.h#L648).   
We can try to find out what high level call brings us here, and compare the performance for the two scenarios of removing the map and using the map.